src/components/LogInForm/LogIn.jsx

```diff
@@ -35,6 +36,17 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
+
+firebase.auth().setPersistence(firebase.auth.Auth.Persistence.SESSION).then(function(user){
+  console.log('hello', user)
+  return firebase.auth().signInWithEmailAndPassword(user.email, user.password);
+})
+.catch(function(error){
+  var errorCode = error.code;
+  var errorMessage = error.message;
+});
+
+
```

Comment:
This is not necessary I think you forgot to remove this code.

Reply:
Oh yeah I was testing it right before I did pull request. Thank you for letting me know.